modules/navier_stokes/doc/content/source/hdgkernels/MassFluxPenaltyIPHDG.md

```diff
@@ -0,0 +1,30 @@
+# MassFluxPenaltyIPHDG
```

Comment:
should probably mention that this goes in the momentum equations not the mass one

Reply:
even though it s called Mass